test/mocks/data/post_comment.dart

```diff
@@ -66,6 +74,7 @@ final List<CommentDetails> unequalComments = [
     content: "content1",
     ownerDisplayName: "username2",
     ownerUsername: "username2username",
+    ownerUserID: const UserIdFirestore(value: "username2ID"),
```

Comment:
To be consitent with the comment, we should have:
```suggestion
    ownerUsername: "username1username",
    ownerUserID: const UserIdFirestore(value: "username1ID"),
```

Reply:
You are right! 

Fixed in :[18b20c1](https://github.com/ProximaEPFL/proxima/pull/348/commits/18b20c1ce2b4d53f54fcebdda746e1683f3668a8)

Thank you 💯